use toggle button group in quicksettings

fm-sys opened the pull request:
fix #10742

eddiemuc commented on main/src/cgeo/geocaching/maps/MapSettingsUtils.java:
```diff
@@ -158,6 +159,50 @@ public void setValue() {
         }
     }
 
+    private static class ToggleButtonWrapper<T> {
```
@fm-sys in my understanding this would replace the ToggleButtonGroup, right? Can you externalize it so it can be used in other places as well (e.g. the filter activity...)? Or should I do that later myself?

fm-sys replied:
First of all, it's a replacement for `ButtonController` (an internal class of `MapSettingsUtils`). But if it looks suitable for you, I can externalize it as well...

eddiemuc replied:
I see now that it is not a view itself.

fm-sys commented:
![grafik](https://user-images.githubusercontent.com/64581222/119866714-638bdf80-bf1d-11eb-8186-d1031f73f275.png)

fm-sys commented:
I removed the "(Current: xxx)" from the Routing mode title, as it was kind of inconsistent and I think the icons are quite clear anyway (we had different icons at the time we implemented the title text). If you tell me, that we definitely need to keep it, I can re-add it for sure...

moving-bits commented:
Leaving out the description for routing mode buttons is ok for me (as I do know their meaning ;-)), but I remember us having some discussion about the icons, which explicitly led to adding that text.

@Lineflyer 
What is your feedback from a support perspective?

Lineflyer commented:
> What is your feedback from a support perspective?

I think we can omit the description of current mode.
I think the buttons are very understandable.

Lineflyer commented:
BTW:
IMHO we could also remove/rephrase the text below regarding installation of BRouter as we now have the internal engine (which is the preferable way IMHO).

eddiemuc commented:
Can we merge this one rather quickly? Right now the buttons are almost unusable in master.
